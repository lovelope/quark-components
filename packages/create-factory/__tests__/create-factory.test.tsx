// @ts-ignore ts2691
import CreateFactory from '../src/create-factory.tsx';

describe('create-factory', () => {
  it('should be defined', () => {
    expect(CreateFactory).toBeDefined();
  });

  it('should be a function', () => {
    expect(typeof CreateFactory === 'function').toBeTruthy();
  });
});
