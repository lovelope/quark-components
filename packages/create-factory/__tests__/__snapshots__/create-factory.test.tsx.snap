// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create-factory components CreateFactory normal 1`] = `
<form
  class="ant-form ant-form-horizontal"
>
  <div
    class="ant-row ant-form-item"
  >
    <div
      class="ant-col ant-form-item-label"
    >
      <label
        class=""
        for="name"
        title="username"
      >
        username
      </label>
    </div>
    <div
      class="ant-col ant-form-item-control-wrapper"
    >
      <div
        class="ant-form-item-control"
      >
        <span
          class="ant-form-item-children"
        >
          <input
            class="ant-input"
            data-__field="[object Object]"
            data-__meta="[object Object]"
            id="name"
            type="text"
            value=""
          />
        </span>
      </div>
    </div>
  </div>
  <div
    class="ant-row ant-form-item"
  >
    <div
      class="ant-col ant-form-item-label"
    >
      <label
        class=""
        for="email"
        title="Eamil"
      >
        Eamil
      </label>
    </div>
    <div
      class="ant-col ant-form-item-control-wrapper"
    >
      <div
        class="ant-form-item-control"
      >
        <span
          class="ant-form-item-children"
        >
          <input
            class="ant-input"
            data-__field="[object Object]"
            data-__meta="[object Object]"
            id="email"
            type="text"
            value=""
          />
        </span>
      </div>
    </div>
  </div>
</form>
`;
